spec/phash_spec.rb

```diff
@@ -5,20 +5,25 @@
   Phash::Image => '**/*.{jpg,png}',
   Phash::Text => '*.txt',
   Phash::Video => '*.mp4',
-}.each do |klass, glob|
+  # mh hash does not perform well on the includes samples,
+  # but as a sanity test this is good enough
+  Phash::MhImage => ['**/*.{jpg,png}', 0.7, 0.5],
```

Comment:
It becomes a very weird test as they suggest that mexican hat image hash is much worse in finding image similarity, maybe it is better to find images for which the hash works well? Also changes to specs are a bit too hackish.